courses/views/v1/__init__.py

```diff
@@ -112,6 +112,10 @@ class CourseFilterSet(django_filters.FilterSet):
     )
 
     def filter_courserun_is_enrollable(self, queryset, _, value):
+        """
+        courserun_is_enrollable filter to narrow down runs that are open for
```

Comment:
Can you just add a definition for what determines if a course run is open for enrollment, and what `value` is doing in regards to narrowing down runs?

Reply:
Something like:

```
Filters courses based on associated courseruns for which: live is True, start_date is not None, enrollment_start is in the past, and enrollment_end is either None or in the future.
@param {boolean} value description about how value is used.
```